org.eclipse.xtext.tests/src-gen/org/eclipse/xtext/parser/terminalrules/services/Bug297105TestLanguageGrammarAccess.java

```diff
@@ -148,14 +148,14 @@
 		private final RuleCall cEXT_INTTerminalRuleCall_2_0 = (RuleCall)cAlternatives_2.eContents().get(0);
 		private final RuleCall cINTTerminalRuleCall_2_1 = (RuleCall)cAlternatives_2.eContents().get(1);
 		
-		//Real hidden():
-		//	INT? '.' (EXT_INT | INT);
+		//Real hidden ():
+		//	INT ? '.' (EXT_INT | INT);
```

Comment:
@ArneDeutsch Unexpected space before the cardinality `?`.

Reply:
Created a unit test, seems to work as expected now.

Reply:
Roger that